package-lock.json

```diff
@@ -14258,39 +14258,58 @@
       "integrity": "sha512-6qE4B9deFBIa9YSpOc9O0Sgc43zTeVYbgDT5veRKSlB2+ZuHNoVVxA1L/ckMUayV9Ay9y7Z/SZCLcGteW9i7bg=="
     },
```

Comment:
Do you need all these changes or just the redux hash update?

Reply:
I update the package.json redux hash and then do a `npm install` and submit whatever npm changed in the package-lock.json file. Is there a better way to do it? I'm just submitting whatever NPM decided to update in the package lock file.

Reply:
Maybe it does, I usually err on the side of doing only the changes explicitly related to the feature and try to avoid lateral changes that I do not understand, that can create side-effects.

Reply:
yea I would avoid checking in these package changes - easy way to fix is 
`git checkout upstream/master package-lock.json` 
Then manually update the mattermost-redux hash inside of `package-lock.json` and commit only those changes

Reply:
Noted. Thanks!

Reply:
you might need to update the hash to the one of the checkin for this PR: https://github.com/mattermost/mattermost-redux/pull/1129/files

Reply:
Done! Thanks